src/Builder.py

```diff
@@ -136,9 +140,10 @@ def __make_move(self, move, node, move_comment):
 
     def __get_cloud_eval(self, fen):
         self.ApiCloudEvalParams['fen'] = fen
-        response = requests.get(
-            self.ApiCloudEvalUrl,
-            params=self.ApiCloudEvalParams)
+        response = requests.get(self.ApiCloudEvalUrl, params=self.ApiCloudEvalParams)
+        while(response.status_code == 429):
+            time.sleep(60)
```

Comment:
Hi Albi ! Well done !

Just a remark: here and on line 113 you are almost copy-pasting the same code (the only differences are the arguments of the get requests).

Do you think it could be possible to define a function or an object dedicated to this task?
Something that can be called like:

```suggestion
response = RequestHandler.TryRequest(api_url, api_params, bad_answers_to_test)
```

where `api_url` can be `self.ApiDbUrl` or `self.ApiCloudEvalUrl` (same logic for `api_params`) and `bad_answers_to_test` can be a list of `status_code` where you know that the answer you got is not the right one.

Of course inside the `TryRequest` function you can implement the while - sleep cycle and whatever you will like to have in a future development.

Last thing: this object can be implemented in a different module that can be used and reused in different parts of this project ;)